sanityv3/schemas/objects/promoTextTile.tsx

```diff
@@ -0,0 +1,104 @@
+import { label } from '@equinor/eds-icons'
+import type { PortableTextBlock, Rule, ValidationContext } from 'sanity'
+import type { ColorSelectorValue } from '../components/ColorSelector'
+import { EdsIcon } from '../../icons'
+import { validateCharCounterEditor } from '../validations/validateCharCounterEditor'
+import type { LinkSelector } from './linkSelector'
+import { configureBlockContent } from '../editors'
+import { getLinkSelectorFields } from './linkSelector'
+
+const ingressBlockContentType = configureBlockContent({
+  h1: false,
+  h2: false,
+  h3: false,
+  h4: false,
+  internalLink: false,
+  externalLink: false,
+  attachment: false,
+  lists: false,
+})
+
+export type PromoTextTile = {
+  _type: 'promoTextTile'
+  linkLabelAsTitle?: boolean
+  link?: LinkSelector
+  isLink?: boolean
+  background?: ColorSelectorValue
+}
+
+const linkFields = getLinkSelectorFields('label', 'isLink')
+
+export default {
+  title: 'Promo text tile',
+  name: 'promoTextTile',
+  type: 'object',
+  fieldsets: [
+    {
+      title: 'Design options',
+      name: 'design',
+      description: 'Some options for design',
+      options: {
+        collapsible: true,
+        collapsed: false,
+      },
+    },
+    {
+      name: 'link',
+      title: 'Link',
+      description: 'Select either an internal link or external URL.',
+    },
+    {
+      name: 'label',
+      title: 'Label',
+      description: 'The label that the link/button should have.',
+    },
+  ],
+  fields: [
+    {
+      name: 'ingress',
+      title: 'Promo Text',
+      description: 'Max 100 characters',
+      type: 'array',
+      of: [ingressBlockContentType],
+      validation: (Rule: Rule) => Rule.custom((value: any) => validateCharCounterEditor(value, 100)),
```

Comment:
value should be typed

Reply:
I see 100 as being too little. Why do we need this limitation? Is this something Nick has requested?

Reply:
this will be revisited when the text can have breaklines, the 100 char limit is a temp limit agreed with nick